src/editors/table.js

```diff
@@ -455,6 +468,12 @@ JSONEditor.defaults.editors.table = JSONEditor.defaults.editors.array.extend({
       e.preventDefault();
       e.stopPropagation();
 
+      if (self.jsoneditor.options.prompt_before_delete === true) {
+        if (confirm("Are you sure you want to remove this node?") === false) {
+          return false;
+        }
+      }
+
```

Comment:
DRY. It's six time the same piece of code, can't we optimize that? CC: @marc7000